src/components/tab-panel/examples/tab-panel-content.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+    Component,
+    h,
+    Prop,
+    EventEmitter,
+    Event,
+    State,
+    Watch,
+} from '@stencil/core';
+import { Tab } from '@limetech/lime-elements';
+
+const LOAD_TIME = 1500;
```

Comment:
LOAD_TIME = 1500 isn't it too much to wait? It is just example 500ms might be ok. Of course this is a personal preference, I leave this for you.

Reply:
1000 it is! :smile: